librosa/decompose.py

```diff
@@ -221,6 +227,13 @@ def hpss(S, kernel_size=31, power=2.0, mask=False):
     mask : bool
         Return the masking matrices instead of components
 
+    margin : float or tuple (margin_harmonic, margin_percussive)
+        margin size(s) for the masks (as described in [2]_)
+
+        - If scalar, the same size is used for both harmonic and percussive.
+        - If iterable, the first value specifies the margin of the
```

Comment:
'iterable' isn't quite right here, because we use explicit indices (0 and 1) rather than `next`ing.  I'd just leave it as tuple.